From a0ce57541f4f22c4a7201de317e603103d6ffdc6 Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppym@gentoo.org>
Date: Sun, 24 Oct 2021 10:04:38 -0400
Subject: [PATCH] channels/rdpei: avoid calling free() on contactPoints array

This was fixed on master as part of
268bc2e8ef7b76887d14d50702bb8446ce15a4c1. Backporting that change seems
like too much for the stable branch.
---
 channels/rdpei/client/rdpei_main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/channels/rdpei/client/rdpei_main.c b/channels/rdpei/client/rdpei_main.c
index f20d58a13..1d95054a0 100644
--- a/channels/rdpei/client/rdpei_main.c
+++ b/channels/rdpei/client/rdpei_main.c
@@ -1386,7 +1386,6 @@ UINT DVCPluginEntry(IDRDYNVC_ENTRY_POINTS* pEntryPoints)
 	return CHANNEL_RC_OK;
 error_out:
 	free(context);
-	free(rdpei->contactPoints);
 	free(rdpei);
 	return error;
 }
-- 
2.33.1

